<ui version="4.0" >
 <class>Form</class>
 <widget class="QWidget" name="Form" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>454</width>
    <height>326</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" >
   <item>
    <layout class="QVBoxLayout" >
     <item>
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <widget class="QLabel" name="label" >
         <property name="text" >
          <string>Delimiter:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" >
        <widget class="QComboBox" name="DelimiterComboBox" >
         <property name="editable" >
          <bool>true</bool>
         </property>
         <item>
          <property name="text" >
           <string>,</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>;</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>\n</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>\t</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="label_2" >
         <property name="text" >
          <string>Line terminator</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="QComboBox" name="LineTerminatorComboBox" >
         <property name="editable" >
          <bool>true</bool>
         </property>
         <item>
          <property name="text" >
           <string>\n</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>\r</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>\t</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>\n\r</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>\r\n</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="0" >
        <widget class="QLabel" name="label_3" >
         <property name="text" >
          <string>Escape char:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" >
        <widget class="QComboBox" name="EscapeCharComboBox" >
         <property name="editable" >
          <bool>true</bool>
         </property>
         <item>
          <property name="text" >
           <string/>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>\</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="0" >
        <widget class="QCheckBox" name="DoubleQuoteCheckBox" >
         <property name="text" >
          <string>Double quote</string>
         </property>
         <property name="checked" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="0" >
        <widget class="QLabel" name="label_4" >
         <property name="text" >
          <string>Quote char:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1" >
        <widget class="QComboBox" name="QuoteCharComboBox" >
         <property name="editable" >
          <bool>true</bool>
         </property>
         <item>
          <property name="text" >
           <string>"</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>'</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="5" column="0" >
        <widget class="QCheckBox" name="QuotingCheckBox" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string>Quoting</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0" >
        <widget class="QCheckBox" name="SkipInitialSpaceCheckBox" >
         <property name="text" >
          <string>Skip initial space</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0" >
        <widget class="QCheckBox" name="StrictCheckBox" >
         <property name="text" >
          <string>Strict</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
